docs/docs/recipes/manage-users/README.md

```diff
@@ -4,8 +4,19 @@ sidebar_position: 6
 
 # 🧑‍🚀 Manage users
 
-TODO: briefly summary
+User management is the core feature for the admin staff.
 
-- Manage users on the Admin Console page
-- Manage users by Logto Core management API
-- Manage user data on the storage
+For example, you want to:
+
+- View the activities of a user whose phone number is "12345678901" to find out when their account has been stolen.
+- Correct the illegal avatar of a user who signed up with the email "pedophilia@exmaple.com".
+- Help a user whose `username` is "forgetful-man" reset the password that they forget.
```

Comment:
`Help xxx reset the password` seems good enough.

Reply:
intended to use `whose `username` is "forgetful-man"` to show we can search the `username` field.

Reply:
Simplified:

- Help a user reset the password that they forget.